epoxy-adapter/src/main/java/com/airbnb/epoxy/BaseEpoxyAdapter.java

```diff
@@ -108,6 +108,15 @@ public void onBindViewHolder(EpoxyViewHolder holder, int position) {
 
   @Override
   public void onBindViewHolder(EpoxyViewHolder holder, int position, List<Object> payloads) {
+    if (holder.itemView instanceof AsyncInflatedView) {
```

Comment:
I think Peter might've mentioned this earlier and I'm not completely familiar with the internals of Epoxy but is there a way where we could avoid these checks? We discussed an implementation where we potentially create an `AsycEpoxyModel` class (similar to how we handled it for Compose interoperability - https://github.com/airbnb/epoxy/blob/master/epoxy-compose/src/main/java/com/airbnb/epoxy/ComposeInterop.kt)